packages/fx-core/src/component/driver/deploy/azure/AzureZipDeployDriver.ts

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureDeployDriver } from "./azureDeployDriver";
+import {
+  AxiosZipDeployResult,
+  AzureUploadConfig,
+  DeployStepArgs,
+} from "../../interface/buildAndDeployArgs";
+import { AzureResourceInfo, DriverContext } from "../../interface/commonArgs";
+import { TokenCredential } from "@azure/core-auth";
+import { IProgressHandler, LogProvider, UserInteraction } from "@microsoft/teamsfx-api";
+import { getLocalizedMessage, ProgressMessages } from "../../../messages";
+import { DeployConstant } from "../../../constant/deployConstant";
+import { createHash } from "crypto";
+import { default as axios } from "axios";
+import { DeployExternalApiCallError } from "../../../error/deployError";
+import { HttpStatusCode } from "../../../constant/commonConstant";
+
+export class AzureZipDeployDriver extends AzureDeployDriver {
+  pattern =
+    /\/subscriptions\/([^\/]*)\/resourceGroups\/([^\/]*)\/providers\/Microsoft.Web\/sites\/([^\/]*)/i;
+  private readonly serviceName: string;
+  protected helpLink = "https://aka.ms/teamsfx-actions/azure-app-service-deploy";
+  protected summaries: string[];
```

Comment:
I see "summaries" and "summaryPrepare" have already been defined in packages/fx-core/src/component/driver/deploy/azure/baseDeployDriver.ts, why do we define it again in this class?